routes/auth.js

```diff
@@ -12,18 +12,18 @@ router.get('/', (req, res) => {
 
 // show register form
 router.get('/register', (req, res) => {
-    res.render('register')
+    res.render('register');
 });
 // handle sign up login
 router.post('/register', (req, res) => {
     const newUser = new User({ username: req.body.username });
     User.register(newUser, req.body.password, (err, user) => {
         if (err) {
-            req.flash('error', err.message);
-            return res.render('register');
+            console.log(err)
```

Comment:
I don't think this `console.log` should be in production-ready code.

Reply:
It will be deleted in the next commit.